### DISCUSSION TITLE
Graph Memo on Raspberry PI

### DISCUSSION THREAD

**Discussion by begiinner** [pvvx/ATC_MiThermometer#416] [Q&A]
Hello,

I am trying (without luck) to download all data through [GraphMemo](https://pvvx.github.io/ATC_MiThermometer/GraphMemo.html)
My ATCs are patched to pvvx custom FW ver. 4.5

My use case is to download data from ATC Thermometer once a week. I don't need to have data online via MQTT or HA.
Problem is to run any solution for dowload data (e.g. to CSV) on my Raspberry PI 4, 2GB, OS Raspbian.
Python solutions neither [GraphMemo](https://pvvx.github.io/ATC_MiThermometer/GraphMemo.html) does not find any of my ATC LYWSD03MMC.

Any ideas how to make it work on Raspberry? Is it caused by not using Google Chrome ?

Thank you for help !

**Comment by pvvx** [comment]
GraphMemo uses [WEB Bluetooth API](https://webbluetoothcg.github.io/web-bluetooth/).
